app/assets/javascripts/govuk_publishing_components/analytics/mailto-link-tracker.js

```diff
@@ -1,37 +1,36 @@
 ;(function (global) {
   'use strict'
 
-  var $ = global.jQuery
   var GOVUK = global.GOVUK || {}
 
   GOVUK.analyticsPlugins = GOVUK.analyticsPlugins || {}
   GOVUK.analyticsPlugins.mailtoLinkTracker = function () {
-    var mailtoLinkSelector = 'a[href^="mailto:"]'
+    document.querySelector('body').addEventListener('click', function (event) {
+      var element = event.target
+      if (element.tagName !== 'A') {
+        element = element.closest('a')
```

Comment:
Would be want to explicitly call a `closest` polyfill? It seems like we could do this via `govuk-frontend` or `govuk_publishing_components`
```
//= require govuk/vendor/polyfills/Element/prototype/closest.js
```
or
```
//= require ../vendor/polyfills/closest.js
```

I'm aware we may already call this polyfill with the other similar scripts, but until we review the polyfilling approach, we might want to explicitly import them where needed.

Reply:
Good question. Turns out that the reason this polyfill was added was because the govuk-frontend one [doesn't support IE8](https://github.com/alphagov/govuk_publishing_components/pull/1645#discussion_r470529237) (according to my past self). I know we don't support IE8 but I think at the time I made the decision on the basis that we still have some IE8 users and I don't want the JS to error for them if we can help it.

Reply:
Ah, yes, I remember why we have the other polyfill, question is: do we want to add it to this script as well?

Reply:
Sorry - I'd completely misunderstood your question, I thought you were recommending we use the govuk-frontend polyfill over the one in the gem, but you were actually suggesting we just add a polyfill (I thought I'd added one already). Apologies, I've added a polyfill now (and updated the changelog).

Reply:
@alex-ju in updating the changelog and rebasing to master I found a new error caused indirectly by recent changes to the metadata component. I've added the change in a separate commit if you want to check before I merge. Including @danacotoran for reference.